Header header

int32 num

int32[] classes_ids
int32[] tracking_ids
int32[] tracking_2d_ids
geometry_msgs/Point[] positions
